### PR TITLE
XWIKI-21370: Header structure uses too many H1

### PR DESCRIPTION
**Pull request by Sereza7**
## Jira
https://jira.xwiki.org/browse/XWIKI-21370
## PR Changes
* Removed H1 from the Dashboard sections 
* Removed H1 from the Panels
## View
vvv Before the PR
21370-beforePR

vvv After the PR
21370-afterPR

We can see by comparing those two that the sections in the dashboard have smaller titles, and that the panel headers are less bold (height is fixed but boldness is linked to the header level ,which went down).
## Note
Since this involves reverting the commit https://github.com/xwiki/xwiki-platform/commit/14547d6acf93ab422c3afc43029d7019ad80a2e4 , I'm leaving this PR as a draft until we can converge to a solution on the forum. This is more of a PoC for now.

### CODE REVIEW

**Comment by manuelleduc** [comment]
> until we can converge to a solution on the forum

@Sereza7 can you share the link to the forum discussion?

**Comment by Sereza7** [comment]
Sorry, it's still not up, I have more thinking to do on this subject before being able to propose a clean and understandable solution. I thought this'd be quick, but with the discussion we had on riot, and needing to put things down in an articulate, it doesn't seem that easy anymore ^^'

**Comment by Sereza7** [comment]
Opened a topic on stack overflow to get input from outside of the community. After looking for a while, I couldn't get any 'hard proof' to use H2s instead of H1s in panels, so I hope I'll get insight from accessibility / usability specialists that might have come upon this issue (or similar) in another tool.
https://stackoverflow.com/questions/77502898/what-would-be-the-recommended-level-for-complementary-headers
:crossed_fingers:

**Comment by Sereza7** [comment]
[Here is the proposal on the forum](https://forum.xwiki.org/t/accessibility-proposal-removing-extra-h1s-from-the-page/13560).

After the changes in https://github.com/xwiki/xwiki-platform/pull/2448/commits/f2214c8bc9e22bf0f4abcfdbb0084bba5c66c37c and https://github.com/xwiki/xwiki-platform/pull/2448/commits/03cde026e5a067c609e16b20e4f6eb5abf8c6b34 , here are how a few more pages are looking like:

21370-afterPRPreferences
21370-afterPRPreferences2
We can see on the screenshot above that the HTML for the preference page only contains one h1 (between panels and preference sections, it contained about 20 before).
21370-afterPRHome
21370-afterPRSandbox
We can see on the screenshot above that in addition to shifting all the heading levels on the Sandbox page, I added an info block to highlight the best practice proposed in the forum topic mentionned at the start of this message.

File: xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml [hunk 1]
```diff
@@ -514,13 +514,13 @@ return XWiki;
   margin-left: 14em;
 }
 
-.column h1 {
+.column h2 {
   font-weight:bold;
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml]: > Don't we have bold headings by default now?\

Yup, removing it in https://github.com/xwiki/xwiki-platform/pull/2448/commits/f56178e9d602f08305ec273e47cf522e70846312

> Also, doesn't this change the appearance?

21370-defaultProfileHeading
^^^ before PR | after PR vvv
21370-afterPRappearance

AFAICS this is the same.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/panels.less [hunk 2]
```diff
@@ -7,7 +7,7 @@
   box-shadow: none;
 }
 
-h1.xwikipaneltitle {
+h2.xwikipaneltitle {
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/panels.less]: https://github.com/search?q=org%3Axwiki%20xwikipaneltitle&type=code
https://extensions.xwiki.org/xwiki/bin/view/Extension/Panel%20Macro
This only place where I can see this class used outside of xwiki-platform is in this macro.
Addressed in https://github.com/xwiki/xwiki-platform/pull/2448/commits/8633ffec91eb0a4a959bbce97cacb3766e75b196

**Pull request merged**